chart/templates/controller-manager.yaml

```diff
@@ -1,237 +1 @@
-    app.kubernetes.io/instance: controller-manager
-    app.kubernetes.io/managed-by: kustomize
-    app.kubernetes.io/name: deployment
-    app.kubernetes.io/part-of: kubestellar
-    control-plane: controller-manager
-  name: kubestellar-controller-manager
-spec:
-  replicas: 1
-  selector:
-    matchLabels:
-      control-plane: controller-manager
-  template:
-    metadata:
-      annotations:
-        kubectl.kubernetes.io/default-container: manager
-      labels:
-        control-plane: controller-manager
-    spec:
-      affinity:
-        nodeAffinity:
-          requiredDuringSchedulingIgnoredDuringExecution:
-            nodeSelectorTerms:
-            - matchExpressions:
-              - key: kubernetes.io/arch
-                operator: In
-                values:
-                - amd64
-                - arm64
-                - ppc64le
-                - s390x
-              - key: kubernetes.io/os
-                operator: In
-                values:
-                - linux
-      containers:
-      - args:
-        - --secure-listen-address=0.0.0.0:8443
-        - --upstream=http://127.0.0.1:8080/
-        - --logtostderr=true
-        - --v=0
-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.13.1
-        name: kube-rbac-proxy
-        ports:
-        - containerPort: 8443
-          name: https
-          protocol: TCP
-        resources:
-          limits:
-            cpu: 500m
-            memory: 128Mi
-          requests:
-            cpu: 5m
-            memory: 64Mi
-        securityContext:
-          allowPrivilegeEscalation: false
-          capabilities:
-            drop:
-            - ALL
-      - args:
-        - --health-probe-bind-address=:8081
-        - --metrics-bind-address=127.0.0.1:8080
-        - --leader-elect
-        - --its-name={{.Values.ITSName}}
-        - --wds-name={{.Values.ControlPlaneName}}
-        - --api-groups={{.Values.APIGroups}}
-        - -v=2
-        image: ghcr.io/kubestellar/kubestellar/controller-manager:tag_is_replaced_in_publishing_workflow
-        imagePullPolicy: IfNotPresent
-        livenessProbe:
-          httpGet:
-            path: /healthz
-            port: 8081
-          initialDelaySeconds: 15
-          periodSeconds: 20
-        name: manager
-        readinessProbe:
-          httpGet:
-            path: /readyz
-            port: 8081
-          initialDelaySeconds: 5
-          periodSeconds: 10
-        resources:
-          limits:
-            cpu: 500m
-            memory: 1Gi
-          requests:
-            cpu: 10m
-            memory: 64Mi
-        securityContext:
-          allowPrivilegeEscalation: false
-          capabilities:
-            drop:
-            - ALL
-      securityContext:
-        runAsNonRoot: true
-      serviceAccountName: kubestellar-controller-manager
-      terminationGracePeriodSeconds: 10
+This file is overwritten by `make chart` (whose results are not kept in Git) and ignored by `make local-chart`.
```

Comment:
I considered that, and thought that leaving an explanation in place would be more helpful.

Reply:
This seems strange. It's as if we will leave such a place holder for all our generated files that are not in git. I don't see the value in this

Reply:
I think that this one is a fairly special case. First, note that currently this file **is** under Git control. More importantly, it is part of a suite of files --- the ones that make up the Helm chart definition. The rest of the suite is in the Git-controlled contents, and the reader will would reasonably wonder about it if this were not here. Finally, note that this is not the only Git-controller place where we have a note saying that it gets replaced.

Reply:
I still strongly believe we should just remove it  and instead put the comment that the file is generate in the docs and/or  in the helm definition. 
However, I will not block the PR because of this. I leave it up to you ...